x-pack/plugins/fleet/common/types/rest_spec/epm.ts

```diff
@@ -46,6 +49,26 @@ export interface GetPackagesResponse {
   response?: PackageList;
 }
 
+interface InstalledPackage {
```

Comment:
This is fairly minimal, and doesn't necessarily align with the advice given, r.e.:

> I do think we should make sure the response body is somewhat aligned with the rest of the Fleet API's patterns though for consistency

Extra properties would have required extra queries to other locations (e.g. the registry), which is why this is so minimal.

Reply:
I think it makes sense to keep the API response minimal to avoid extra queries, we can add more fields later if required.